### PR TITLE
Add metrics about the powchain service

### PR DESCRIPTION
**Pull request by prestonvanloon**
This will give us some visibility into the status of the beacon chain in a remote deployment. 

See these metrics at http://localhost:8080/metrics

### CODE REVIEW

File: beacon-chain/powchain/service.go [hunk 1]
```diff
@@ -26,6 +28,25 @@ import (
...
+		Name: "powchain_valid_deposits_received",
+		Help: "The number of valid deposits received in the deposit contract",
+	})
+	totalDepositsCount = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "powchain_deposit_logs",
+		Help: "The total number of deposits received in the deposit contract",
+	})
+	chainStartCount = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "powchain_chainstart_logs",
+		Help: "The number of chainstart logs received from the deposit contract",
```
Comment (terenc3t) [inline beacon-chain/powchain/service.go]: only one chain start will happen(it is boolean) , shouldnt this be more in the line of ‘Has chain start log filed?’

Comment (prestonvanloon) [reply beacon-chain/powchain/service.go]: It should only happen once, but if it happens more than once, then that is a very interesting metric to alert. Don't you think? :)

Comment (prestonvanloon) [reply beacon-chain/powchain/service.go]: Think: reorg, or something

Comment (terenc3t) [reply beacon-chain/powchain/service.go]: True true. Let's hope that never happens 🙏🏼

File: beacon-chain/powchain/service.go [hunk 2]
```diff
@@ -26,6 +28,25 @@ import (
...
+	})
+	blockNumberGuage = promauto.NewGauge(prometheus.GaugeOpts{
```
Comment (nisdas) [inline beacon-chain/powchain/service.go]: `blockNumberGauge`

File: beacon-chain/powchain/service.go [hunk 3]
```diff
@@ -225,6 +246,7 @@ func (w *Web3Service) ProcessLog(VRClog gethTypes.Log) {
 // the ETH1.0 chain by trying to ascertain which participant deposited
 // in the contract.
 func (w *Web3Service) ProcessDepositLog(VRClog gethTypes.Log) {
+	totalDepositsCount.Inc()
```
Comment (nisdas) [inline beacon-chain/powchain/service.go]: the `Inc()` method is to increment this , right ?

File: beacon-chain/powchain/service.go (hunk 2 above)
Comment (prestonvanloon) [reply beacon-chain/powchain/service.go]: Nice catch

File: beacon-chain/powchain/service.go (hunk 3 above)
Comment (prestonvanloon) [reply beacon-chain/powchain/service.go]: Yep!

**Pull request merged**